README.md

```
@@ -325,371 +236,84 @@
+   // Execution continues after fetch_data completes
+   ```
+
+2. **Async First, Await Later (`.async` → `.await`)**  
+   Start the process and receive a `Future`, then wait for it when you need:
+   ```pseudo
+   let future = fetch_data("https://example.org").async;
+   // ... do other work ...
+   let result = future.await;
+   ```
+
+#### Benefits
```

Markdownlint (0.37.0)

Expected: 1; Actual: 0; Below
Headings should be surrounded by blank lines

(MD022, blanks-around-headings)